lighthouse-core/audits/service-worker.js

```diff
@@ -73,23 +73,31 @@ class ServiceWorker extends Audit {
    * @param {Array<LH.Crdp.ServiceWorker.ServiceWorkerVersion>} matchingSWVersions
    * @param {Array<LH.Crdp.ServiceWorker.ServiceWorkerRegistration>} registrations
    * @param {URL} pageUrl
-   * @return {string|undefined}
+   * @return {Array<string> | undefined}
    */
   static getControllingScopeUrl(matchingSWVersions, registrations, pageUrl) {
```

Comment:
now that this returns more than just the scope url we should rename it... perhaps `getControllingServiceWorker`